pkg/crc/api/handlers.go

```diff
@@ -214,9 +214,12 @@ func (h *Handler) UnsetConfig(c *context) error {
 }
 
 func (h *Handler) GetConfig(c *context) error {
+	queries := c.url.Query()
 	var req client.GetOrUnsetConfigRequest
-	if err := c.Bind(&req); err != nil {
-		return err
+	if len(queries) != 0 {
+		for key := range queries {
+			req.Properties = append(req.Properties, key)
+		}
 	}
```

Comment:
You have a check for `req.Properties` length just below this, I don't think the `len(queries)` check is needed, if `queries` is empty, `for key := range queries` will loop 0 times, and `len(req.Properties)` will be 0